### PR TITLE
Add angular material library

### PR DESCRIPTION
**Pull request by Johngtka**
Add angular material library to the application

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,10 +12,12 @@
   "private": true,
   "dependencies": {
     "@angular/animations": "^15.1.0",
+    "@angular/cdk": "^15.1.2",
```
Comment (macrog) [inline package.json:15]: what is this for?

Comment (Johngtka) [reply package.json:15]: i dont know I think it was added along with angular materials

File: src/app/app.module.ts [hunk 2]
```diff
@@ -5,7 +5,8 @@ import { HttpClient, HttpClientModule } from '@angular/common/http'
 import { TranslateModule, TranslateLoader } from '@ngx-translate/core'
 import { TranslateHttpLoader } from '@ngx-translate/http-loader'
 
-import { AppComponent } from './app.component'
+import { AppComponent } from './app.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations'
```
Comment (macrog) [inline src/app/app.module.ts:9]: could you move the import to the second block of imports, 3rd party libs please

File: src/custom-theme.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
+
```
Comment (macrog) [inline src/custom-theme.scss:1]: what is this? do we need a custom-theme?

**Comment by macrog** [comment]
I have made a quick test by including autocomplete and it works, good job @Johngtka !!  please fix the imports, check if we need the custom-theme and let me know.

image

**Comment by macrog** [comment]
@Johngtka please provide some [description](https://github.com/Johngtka/Biomagnthick-app/pull/6#issue-1568079566) and title to this PR it looks sad, and also write you commit message proposal for squash and merge!?

File: src/custom-theme.scss (hunk 3 above)
Comment (Johngtka) [reply src/custom-theme.scss:1]: i dont know, maybe we will use this

File: src/app/app.module.ts (hunk 2 above)
Comment (Johngtka) [reply src/app/app.module.ts:9]: explain

**Comment by Johngtka** [comment]
how are you test autocomplete and how i can test this

File: src/app/app.module.ts (hunk 2 above)
Comment (macrog) [reply src/app/app.module.ts:9]: what we have talked yesterday, when I show you 3 blocks of imports

**Comment by macrog** [comment]
squash and merge:

chore: Add angular material support to the application (#6)

**Review by macrog — APPROVED** [review]
🔨

**Pull request merged**